cvmfs/util/posix.cc

```diff
@@ -1630,6 +1630,14 @@ bool SafeWriteToFile(const std::string &content,
   return retval;
 }
 
+/**
+ * Returns current time in nanoseconds
+ * It is not guarantee to be monotomic increasing
+ */
+uint64_t GetTime() {
```

Comment:
--> `GetRealTimeNs` -- or, the user code could use directly the `platform_...` functions.